onnx_chainer/context.py

```diff
@@ -26,15 +26,31 @@ def __init__(self, model):
     def get_name(self, variable):
         str_id = id(variable)
         if str_id in self.name_list:
-            return self.name_list[str_id]
+            return self.name_list[str_id][0]
         else:
             new_name = 'v{}'.format(len(self.name_list))
             self.set_name(variable, new_name)
             return new_name
 
-    def set_name(self, variable, name):
+    def set_name(self, variable, name, pinned=False):
+        """Set ONNX node name
+
+        Arguments:
+            variable (var): target variable
+            name (str): name to be exported as ONNX node name
+            pinned (bool): if don't want to change the name on subsequent
+                process, set ``True``. Enable to check the target variable
+                is pinned or not by using ``is_pinned``.
+        """
+
```

Comment:
I think it's invalid to call `set_name` for pinned names? How about `assert str_id not in self.name_list or not self.name_list[str_id][1]`?